### ISSUE TITLE
Chinese font is missing

### ISSUE THREAD

**Issue by itcuihao** [signintech/gopdf#68]
Hi:
 I use gopdf.

1. 

import:"您好，为了你"；

2. 

 export:"您好，    了你"

Why is this？

code:

>
	pdf := gopdf.GoPdf{}
	pdf.Start(gopdf.Config{PageSize: gopdf.Rect{W: 595.28, H: 841.89}}) //595.28, 841.89 = A4
	pdf.AddPage()
	var err error

	err = pdf.AddTTFFont("HDZB_5", "./ttf/wts11.ttf")
	if err != nil {
		log.Print(err.Error())
		return
	}
	//china
	err = pdf.SetFont("HDZB_5", "", 14)
	if err != nil {
		log.Println(2)
		log.Printf(err.Error())
		return
	}
	commitMessage := `  您好，为了您的`
	log.Println(commitMessage)
	err = pdf.Cell(nil, commitMessage)
	if err != nil {
		log.Println(3)
		log.Printf(err.Error())
		return
	}
	pdf.Br(20)
	pdf.WritePdf("hello.pdf")

**Comment by itcuihao** [comment]
![image](https://user-images.githubusercontent.com/9942270/30142841-6ae930a8-93b5-11e7-8abf-ac2394cc8e20.png)

**Comment by itcuihao** [comment]
thanks!

**Comment by oneplus1000** [comment]
I think file 'wts11.ttf' not contain glyph of character "为" (4e3a). 
![screenshot at 2017-09-07 17-29-59](https://user-images.githubusercontent.com/1155104/30159521-0e5b8000-93f3-11e7-919b-03c7084e3f8d.png)

**Comment by itcuihao** [comment]
@oneplus1000 Yes.I can change the stsong.ttf file to display Chinese characters.Thank you for your reply.

**Comment by tjl-cmd** [comment]
> 嗨： 我使用 gopdf。
> 
> 
> 导入：“你好，为了你”；
> 
> 
> export:"你好，了你"
> 
> 为什么是这样？
> 
> 代码：
> 
> > 
> 
> ```
> pdf := gopdf.GoPdf{}
> pdf.Start(gopdf.Config{PageSize: gopdf.Rect{W: 595.28, H: 841.89}}) //595.28, 841.89 = A4
> pdf.AddPage()
> var err error
> 
> err = pdf.AddTTFFont("HDZB_5", "./ttf/wts11.ttf")
> if err != nil {
> 	log.Print(err.Error())
> 	return
> }
> //china
> err = pdf.SetFont("HDZB_5", "", 14)
> if err != nil {
> 	log.Println(2)
> 	log.Printf(err.Error())
> 	return
> }
> commitMessage := `  您好，为了您的`
> log.Println(commitMessage)
> err = pdf.Cell(nil, commitMessage)
> if err != nil {
> 	log.Println(3)
> 	log.Printf(err.Error())
> 	return
> }
> pdf.Br(20)
> pdf.WritePdf("hello.pdf")
> ```

可以发一份字体给我吗？？我现在遇到中文空白无法显示出来的问题

**Comment by vantaboard** [comment]
@oneplus1000 can you close this issue if it has been resolved?
